app/build.gradle

```diff
@@ -108,10 +108,11 @@ dependencies {
   implementation project(path: ':common:pages')
 
   implementation deps.kotlin.stdlib
-  implementation "androidx.appcompat:appcompat:${androidxVersion}"
-  implementation "androidx.media:media:${androidxVersion}"
-  implementation "androidx.recyclerview:recyclerview:${androidxVersion}"
-  implementation "com.google.android.material:material:${androidxVersion}"
+  implementation "androidx.appcompat:appcompat:${supportLibVersion}"
+  implementation "androidx.media:media:${supportLibVersion}"
+  implementation "androidx.recyclerview:recyclerview:${supportLibVersion}"
+  implementation 'androidx.preference:preference:1.0.0'
```

Comment:
```suggestion
  implementation "androidx.preference:preference:${androidxVersion}"
```

Reply:
personally I prefer we don't use single constant value for all android version, because unlike android support library, androidx library version can be different between packages:

>Unlike the Support Library, AndroidX packages are separately maintained and updated. The androidx packages use strict Semantic Versioning starting with version 1.0.0. You can update AndroidX libraries in your project independently.

reference: https://developer.android.com/jetpack/androidx/